uxarray/remap/inverse_distance_weighted.py

```diff
@@ -148,7 +148,6 @@ def _inverse_distance_weighted_remap_uxda(
     uxda_remap = uxarray.core.dataarray.UxDataArray(
         data=destination_data,
         name=source_uxda.name,
-        coords=source_uxda.coords,
```

Comment:
@philipc2 Here's my solution to preserve any non-spatial coordinates (like time or ensemble_member) while dropping the spatial one:

Copy `source_uxda.coords` to a new `xarray.Coordinates` object. Then delete the key associated with the spatial dimension. Then use that new object as the coords argument to `uxarray.core.dataarray.UxDataArray`.

```
    destination_coords = source_uxda.coords
    del(destination_coords[destination_dim])

    # construct data array for remapping variable
    uxda_remap = uxarray.core.dataarray.UxDataArray(
        data=destination_data,
        name=source_uxda.name,
        coords=destination_coords,
```
Does this make sense?

Reply:
@philipc2 Thanks .  Just to make sure I understand, does the _preserve_coordinates function slice along the spatial coordinate and return the sliced spatial coordinate (along with the non-spatial coordinates)? If so, that's great. I don't think my solution did that. My solution simply deleted the spatial coordinate. Retaining the properly-sliced spatial coordinate is good.  
Therefore, I don't think we really need a `dims_to_drop` option.  When I dropped it in my suggested code, I was just being lazy. I couldn't think of a way to provide the sliced spatial coordinates. We don't want to completely drop the spatial dimension, just slice it.  So could the `dims_to_drop` option be removed?

Reply:
I've updated the logic in the `UxDataArray.isel()` function, which should now correctly handle slicing the spatial dimensions.

As for the remapping, I've re-added the deletion approach you mentioned, with a few extra checks.